docs/evm/near-eth-rpc.md

```diff
@@ -0,0 +1,12 @@
+---
+id: near-eth-rpc
+title: Proxy Ethereum RPC Server
+sidebar_label: Proxy RPC Server
+---
+
+There is a <a href="https://github.com/near/near-eth-rpc" target="_blank">proxy server for the Ethereum JSON RPC to the NEAR EVM</a>.
```

Comment:
"Web3 JSON-RPC API"

Reply:
I think you mean line 4 here for the `sidebar_label` ✅

Reply:
I think this was actually about line 7. As in, "Ethereum JSON RPC" => "Web3 JSON-RPC API".